src/controllers/aplication-controller.js

```diff
@@ -0,0 +1,10 @@
+import App from '../models/Application-model';
+
+/**
```

Comment:
## Describe 

You need to describe before all the @ params and returns what your function is about